poetry.lock

```diff
@@ -1,148 +1,387 @@
+# This file is automatically @generated by Poetry and should not be changed by hand.
+
 [[package]]
```

Comment:
Could cause conflicts with #6, will see once that's merged

Reply:
Will rebase on top of develop when #6 is merged as necessary.